packages/ui/src/components/Catalog/tag-color-resolver.ts

```diff
@@ -0,0 +1,12 @@
+export type COLOR = 'blue' | 'cyan' | 'green' | 'orange' | 'purple' | 'red' | 'grey' | 'gold';
+
+export const getTagColor = (tag: string): COLOR => {
```

Comment:
Maybe we could spare this function and achieve the same using `CSS` only if we set the `tag` itself to a `CSS` class.